modules/osdk-golang-run-in-cluster.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-run-in-cluster_{context}"]
+= Running as a deployment inside the cluster
+
+You can run your Operator project as a deployment inside of your cluster.
+
+.Procedure
+
+. *Build and push the image.*
```

Comment:
I'd remove the bold highlighting from this module - the rest of the modules in the assembly don't use it, and it looks weird in context.